queries/hcl/highlights.scm

```diff
@@ -0,0 +1,72 @@
+; highlights.scm
+
+[
+  "!"
+  "\*"
+  "/"
+  "%"
+  "\+"
+  "-"
+  ">"
+  ">="
+  "<"
+  "<="
+  "=="
+  "!="
+  "&&"
+  "||"
+] @operator
+
+[
+  "{"
+  "}"
+  "["
+  "]"
+  "("
+  ")"
+  "[*]"
+  ".*"
+] @specialChar
```

Comment:
These should be `punctuation.bracket`

Reply:
hey, thank you! i changed it just now. I think (hope) all capture groups are valid now.